src/Components/CartCards/BillCardForCart.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { useFilter } from "../../Context/index";
+import { calculatePrice, discountPrice } from "./CardUtils";
+import { AiOutlineMinus } from "react-icons/ai";
+export const BillCardForCart = () => {
+  const { state } = useFilter();
+  const { cartList } = state;
+  let price = cartList.length > 0 ? calculatePrice(cartList) + 199 : 0;
+  let discountObj = cartList.length > 0 ? discountPrice(cartList) : 0;
```

Comment:
Suggestion: You may use [nullish operator `??`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) here.